help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,180 @@
+---
+keywords: crm;CRM;crm destinations;salesforce crm;salesforce crm destination
+title: Salesforce CRM connection
+description: The Salesforce CRM destination allows you to export your account data and activate it within Salesforce CRM for your business needs.
+---
+
+# [!DNL Salesforce CRM] connection
+
+## Overview {#overview}
+
+[Salesforce CRM](https://www.salesforce.com/) is a popular Customer Relationship Management (CRM) platform.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts), which allows you to update identities within a segment into Salesforce CRM.
+
+## Destination specifics {#specifics}
+
+Note the following details that are specific to the [!DNL Salesforce CRM] destination:
+
+* [!DNL Adobe Experience Platform] segments are exported to [!DNL Salesforce CRM] using a custom `AdobeExperiencePlatformSegments` field. Instructions are further below, in the [Prerequisites](#prerequisites) section.
+
+* Salesforce CRM uses OAuth 2 with Password Grant as an authentication mechanism to communicate with the Salesforce REST API. Instructions to authenticate to your Salesforce CRM instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, I can deliver personalized experiences to my users, based on attributes from their Adobe Experience Platform profiles, build segments from my offline data, and send these segments to Salesforce CRM, to display in my users’ feeds, as soon as segments and profiles are updated in Adobe Experience Platform.
```

Comment:
Sorry, I missed this earlier. Can we please rephrase from 1st person to 3rd person? Also, as a general rule, let's please try to break up long sentences into smaller, separate sentences. After three or four commas, a sentence becomes difficult to follow. See an improved example below:

As a marketer, you can deliver personalized experiences to your users, based on attributes from their Adobe Experience Platform profiles. You can  build segments from your offline data and send these segments to Salesforce CRM, to display in your users’ feeds as soon as segments and profiles are updated in Experience Platform.

Reply:
Apologies, had forgotten the 1st person to 3rd person. Comma's gotcha.